recipe/meta.yaml

```diff
@@ -1,53 +1,57 @@
 {% set name = "pytest-doctestplus" %}
-{% set version = "0.11.1" %}
-{% set git_url = "https://github.com/astropy/pytest-doctestplus" %}
+{% set version = "1.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b7a0aeb79b85ee81a3c72c49019b4bfeb57fa920abaa6c17ba8be3be9c5290f1
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 709ad23ea98da9a835ace0a4365c85371c376e000f2860f30de6df3a6f00728a
 
 build:
-  noarch: python
+  skip: True  # [py<38]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
-    - setuptools >=30.3.0
+    - setuptools
     - setuptools_scm
     - wheel
   run:
-    - python >=3.7
+    - python
     - pytest >=4.6
     - setuptools >=30.3.0
```

Comment:
```suggestion
```
They drop `setuptools` as a dependency, see https://github.com/scientific-python/pytest-doctestplus/blob/v1.3.0/setup.cfg#L35-L37 and https://github.com/scientific-python/pytest-doctestplus/blob/v1.3.0/CHANGES.rst#130-2024-11-25